hotsos/core/plugins/storage/ceph.py

```diff
@@ -247,12 +247,25 @@ def crushmap_equal_buckets_pretty(self):
         if unequal:
             return ", ".join(unequal)
 
+    @cached_property
+    def autoscaler_disabled_for_any_pool(self):
+        if not self.ceph_report:
+            return True
+
+        pools = self.ceph_report['osdmap']['pools']
+        for pool in pools:
+            if (pool.get('pg_autoscale_mode') is None) or \
+                    pool['pg_autoscale_mode'] != 'on':
+                return True
+        return False
+
 
 class CephCluster(object):
     OSD_META_LIMIT_PERCENT = 5
     OSD_PG_MAX_LIMIT = 500
     OSD_PG_OPTIMAL_NUM_MAX = 200
     OSD_PG_OPTIMAL_NUM_MIN = 50
+    POOL_USAGE_LIMIT_PERCENT = 2
```

Comment:
This "2%" is arbitrary - sometimes FE run tests/benchmarks and there's the device_health_metrics that's there as well. If handover takes a few weeks then there might be some data in one of these pools as well.

So realistically we can't expect "0%" utilisation in any cluster.

Reply:
The 2% check seems reasonable enough. This constant was a bit confusing for me. I'd recommend adding a comment here to clarify the intent, or refactor this to align with the scenario name change above.